cfg/rh-1.0/policies.yaml

```diff
@@ -13,7 +13,7 @@ groups:
         type: "manual"
         audit: |
           #To get a list of users and service accounts with the cluster-admin role
-          oc get clusterrolebindings -o=customcolumns=NAME:.metadata.name,ROLE:.roleRef.name,SUBJECT:.subjects[*].kind |
+          oc get clusterrolebindings -o=custom-columns=NAME:.metadata.name,ROLE:.roleRef.name,SUBJECT:.subjects[*].kind |
```

Comment:
This was a typo in the benchmark. I was comparing both the standards when I identified this. The type is marked as `manual`, so wasn't caught earlier.